### ISSUE TITLE
PANR - HPC issues

### ISSUE THREAD

**Issue by ewagajda** [ewagajda/OMICS-tools#2]
```
> source R-3.2.2
> R
```

1.

```
> library(PANR)

> pan<-new("PAN", bm1=bm1)
> pan<-infer(pan, para=list(type="SNR", log=TRUE, sign=TRUE,  
cutoff=log(5)), filter=FALSE, verbose=TRUE)
> data(Bakal2007Cluster)
> pan<-buildPAN(pan, engine="RedeR", para=list(nodeColor=nodeColor, hideNeg=TRUE))
```

> --use default args:  nodeSumCols, nodeSumMethod
> -building RedeR graph ...
> -finished building RedeR graph ...

`> library(RedeR)`

> ***This is RedeR 1.18.1! For a quick start, please type 'vignette('RedeR')'.
>    Supporting information is available at Genome Biology 13:R29, 2012,
>    (doi:10.1186/gb-2012-13-4-r29).

`> viewPAN(pan, what="graph")`
................................................................................

> The Java interface is not responding to initialization!
> Please, check whether Java is already installed in your machine (JRE version>=6).
> For a general diagnosis, re-run the 'calld' function with 'checkcalls=TRUE', for example: > calld(rdp, checkcalls=TRUE)
> Please report any eventual error message to us <mauro.a.castro at gmail.com>

2.

```
> inds<-sigModules(pan,pValCutoff=0.01, minSize=3, maxSize=100, sortby="pval", 
decreasing=FALSE)
> viewPAN(pan,what="pvclustModule", moduleID=inds)
```

................................................................................

> The Java interface is not responding to initialization!
> Please, check whether Java is already installed in your machine (JRE version>=6).
> For a general diagnosis, re-run the 'calld' function with 'checkcalls=TRUE', for example: > calld(rdp, checkcalls=TRUE)
> Please report any eventual error message to us <mauro.a.castro at gmail.com>
> There were 50 or more warnings (use warnings() to see the first 50)

`> warnings()`

> Warning messages:
> 1: In .Call("R_igraph_subgraph", graph, as.igraph.vs(graph,  ... :
>   At structural_properties.c:1945 :igraph_subgraph is deprecated from igraph 0.6, use igraph_induced_subgraph instead

3.
`> viewNestedModules(pan,pValCutoff=0.05,minSize=3,maxSize=100)`
................................................................................

> The Java interface is not responding to initialization!
> Please, check whether Java is already installed in your machine (JRE version>=6).
> For a general diagnosis, re-run the 'calld' function with 'checkcalls=TRUE', for example:> calld(rdp, checkcalls=TRUE)
> Please report any eventual error message to us <mauro.a.castro at gmail.com>

**Comment by wjurkowski** [comment]
I got an error whilst installing PANR. Did you load libcurl prior to installing PANR and launching it later?
source libcurl-7.32.0

**Comment by wjurkowski** [comment]
On the other hand the interactive visualisation wont work over the net. Are other functions of PANR working?

**Comment by wjurkowski** [comment]
if the problem persist let's test it locally

**Comment by ewagajda** [comment]
I didn't load libcurl prior to installing PANR. 
This is my log from PANR instalation

```
ssh software
source R-3.2.2
R 
source("http://bioconductor.org/biocLite.R")
biocLite("PANR")

```

> BioC_mirror: https://bioconductor.org
> Using Bioconductor 3.2 (BiocInstaller 1.20.3), R 3.2.2 (2015-08-14).
> Installing package(s) ‘PANR’
> trying URL 'https://bioconductor.org/packages/3.2/bioc/src/contrib/PANR_1.16.0.tar.gz'
> Content type 'application/x-gzip' length 2074406 bytes (2.0 MB)
> 
> downloaded 2.0 MB
> - installing _source_ package ‘PANR’ ...
>   *\* R
>   *\* data
>   *\* inst
>   *\* preparing package for lazy loading
>   *\* help
>   **\* installing help indices
>   *\* building package indices
>   *\* installing vignettes
>   *\* testing if installed package can be loaded
> - DONE (PANR)
> 
> The downloaded source packages are in
>         ‘/tmp/RtmpGY64VW/downloaded_packages’
> Old packages: 'robustbase', 'ape', 'BH', 'Cairo', 'corrplot', 'DBI',
>   'devtools', 'evaluate', 'fields', 'formatR', 'git2r', 'highr', 'Hmisc',
>   'jsonlite', 'knitr', 'Matrix', 'nlme', 'openssl', 'phyloseq', 'plotrix',
>   'Rcpp', 'RcppArmadillo', 'RCurl', 'scatterplot3d', 'stringi', 'survival',
>   'testthat'
> Update all/some/none? [a/s/n]: n

**Comment by ewagajda** [comment]
Yes, other functions are working, it's just the graphics that don't.
